R/PRISMA_flowdiagram.R

```diff
@@ -734,6 +707,77 @@ sr_flow_interactive <- function(plot,
 #' PRISMA_save(plot, format = 'pdf')
 #' }
 #' @export
-PRISMA_save <- function(plotobj){
-  htmlwidgets::saveWidget(plotobj, file="PRISMA2020_flowdiagram.html")
+PRISMA_save <- function(plotobj, format = 'HTML', filename = 'PRISMA2020_flowdiagram'){
```

Comment:
Ideally, there would be some functionality in `PRISMA_save()` that would enable the function to guess the format if a file extension is provided to `filename` (using e.g. `file.ext()`), similar to how `ggsave()` works. With `ggsave()`, you can either include the file extension in the filename or specify the device.

https://ggplot2.tidyverse.org/reference/ggsave.html

Note that this would probably require the default value of `format` to be `NULL`.

----

Additionally, my personal preference is for `filename` to be the second argument - but happy for you to disagree with this!

Reply:
I'm indifferent so will change